share/migrations/0034_celery_upgrade.py

```diff
@@ -0,0 +1,85 @@
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('share', '0033_merge_20170509_1710'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CeleryTaskResult',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('correlation_id', models.TextField(blank=True)),
+                ('status', models.CharField(choices=[('FAILURE', 'FAILURE'), ('PENDING', 'PENDING'), ('RECEIVED', 'RECEIVED'), ('RETRY', 'RETRY'), ('REVOKED', 'REVOKED'), ('STARTED', 'STARTED'), ('SUCCESS', 'SUCCESS')], db_index=True, default='PENDING', max_length=50)),
+                ('task_id', models.UUIDField(db_index=True, unique=True)),
+                ('meta', share.models.fields.DateTimeAwareJSONField(editable=False, null=True)),
+                ('result', share.models.fields.DateTimeAwareJSONField(editable=False, null=True)),
+                ('task_name', models.TextField(blank=True, db_index=True, editable=False, null=True)),
+                ('traceback', models.TextField(blank=True, editable=False, null=True)),
+                ('date_created', models.DateTimeField(auto_now_add=True)),
+                ('date_modified', models.DateTimeField(auto_now=True, db_index=True)),
+                ('share_version', models.TextField(default=share.models.logs.get_share_version, editable=False)),
+            ],
+            options={
+                'verbose_name': 'Celery Task Result',
+                'verbose_name_plural': 'Celery Task Results',
+            },
+        ),
+        # ???
+        # migrations.AlterIndexTogether(
+        #     name='celerytask',
+        #     index_together=set([]),
+        # ),
+        migrations.RemoveField(
+            model_name='celerytask',
+            name='provider',
+        ),
+        migrations.RemoveField(
+            model_name='celerytask',
+            name='started_by',
+        ),
+        migrations.DeleteModel(
+            name='CeleryProviderTask',
+        ),
+        migrations.AddField(
+            model_name='sourceconfig',
+            name='backharvesting',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='sourceconfig',
+            name='harvest_after',
+            field=models.TimeField(default='02:00'),
+        ),
+        migrations.AddField(
+            model_name='sourceconfig',
+            name='harvest_interval',
+            field=models.DurationField(default='1 day'),
+        ),
+        migrations.AlterField(
+            model_name='harvestlog',
+            name='status',
+            field=models.IntegerField(choices=[(0, 'Enqueued'), (1, 'In Progress'), (2, 'Failed'), (3, 'Succeeded'), (4, 'Rescheduled'), (6, 'Forced'), (7, 'Skipped'), (8, 'Retrying'), (9, 'Cancelled')], db_index=True, default=0),
+        ),
+        migrations.AlterField(
```

Comment:
I think this should be remove/add instead of alter -- migrating gives me an error:
`psycopg2.IntegrityError: insert or update on table "share_normalizeddata_tasks" violates foreign key constraint "share_normalizeddata_celerytaskresult_id_8bea2c69_fk_share_cel"
DETAIL:  Key (celerytaskresult_id)=(4) is not present in table "share_celerytaskresult".`